Re-import navigation-api WPT

nt1m opened the pull request:
#### 16b3b33cc102e53861c0a3bd6c6ab7160761af86
<pre>
Re-import navigation-api WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=299105">https://bugs.webkit.org/show_bug.cgi?id=299105</a>
<a href="https://rdar.apple.com/160872179">rdar://160872179</a>

Reviewed by Rupin Mittal.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/a1ecbd2950e6a24c6b256e3f96a4226941c07d96">https://github.com/web-platform-tests/wpt/commit/a1ecbd2950e6a24c6b256e3f96a4226941c07d96</a>

* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/history-back-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/navigation-back-forward-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/navigation-navigate-replace-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/navigation-navigate-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/cross-origin-traversal-redirect.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-handler-throws.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-multiple-times-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-popstate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-after-detach.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-dynamic-index.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-noop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-navigation-navigate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-navigation-navigate.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-in-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-back-same-document-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-in-iframe-same-document-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-navigates-top-and-same-doc-child-and-cross-doc-child.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-same-document-preventDefault-multiple-windows.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-top-cancels-cross-document-child.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/replaceState-inside-back-handler.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/same-url-replace-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/same-url-replace-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-history-replaceState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-initial-about-blank.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-push-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-replace-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-same-document-then-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse-not-in-entries.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse-then-clobber.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/after-detach.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entries-after-blank-navigation-from-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/resources/opaque-origin-page.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/back-forward-multiple-frames.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/disambigaute-traverseTo-forward-multiple.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/forward-to-pruned-entry.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/navigate-history-push-same-url-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/navigate-history-push-same-url.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-already-detached.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-beforeunload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-intercept-rejected.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-already-detached.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-beforeunload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-intercept-rejected.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-intercept-interrupted.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-interrupted-within-onnavigate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-interrupted.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-javascript-url-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-javascript-url.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-push-initial-about-blank.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-push-javascript-url-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-push-javascript-url.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/resources/back-forward-opaque-origin-page.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-cross-document-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-cross-document-before-navigate-event.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-same-document-before-navigate-event.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-intercept-rejected.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-repeated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/traverseTo-multiple-steps.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/traverseTo-navigates-multiple-iframes.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/anchor-download-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/anchor-download-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/back-same-document-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/back-same-document-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/back-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/currententrychange-before-popstate-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/intercept-async.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-canceled.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-double-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-intercept-reentrant.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-canceled.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-double-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-in-transition-finished.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-stop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reentrant.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-canceled.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-no-popstate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-options.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push-changed-to-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace-changed-to-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-throws.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-commit-new-navigation-before-commit.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-window-stop-before-commit.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-uncancelable.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-window-stop-before-commit.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/resources/helpers.mjs: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/resources/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-basic.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-change-history-scroll-restoration-during-promise.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-explicit-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-intercept-handler-modifies.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-timing.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-basic.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-immediate-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-after-dispatch.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-after-resolve.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-fragment-does-not-exist.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-in-precommit-handler.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-repeated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-resets-when-no-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/scroll-after-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/scroll-without-intercept.html:

Canonical link: <a href="https://commits.webkit.org/300203@main">https://commits.webkit.org/300203@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f7f2929d1f2b567cc38166fa2cd49770a08fcdff

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/121594 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/41291 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/31950 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/128125 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/73684 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/f8ef8a69-9899-4a7f-a23a-b4cd5d608ec6) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/41993 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/49873 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/92410 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61452 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/2f249483-2ba7-4aa5-adbf-1b74edfcbfce) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/124546 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/33525 "Passed tests") | [❌ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/108938 "Found 1 new API test failure: TestIPC.ConnectionTest/ConnectionRunLoopTest.SendAsyncAndInvalidate/ClientIsA (failure)") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/73072 "Found 1 new API test failure: WPE/TestWebKitWebContext:/webkit/WebKitWebContext/no-web-process-leak (failure)") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/40ee6b09-ca5d-4a0f-a0ad-0cfb3e373a21) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/32547 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/27082 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/71627 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/103029 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/27251 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/130945 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/48528 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/36929 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/100999 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/48897 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/105134 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/100888 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/46257 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/24349 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/45231 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/19279 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/48381 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/47854 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/51205 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/49545 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

RupinMittal reviewed (approved):
LGTM. We should make sure we don't need to update any expectations before landing.

nt1m committed:
Re-import navigation-api WPT
https://bugs.webkit.org/show_bug.cgi?id=299105
rdar://160872179

Reviewed by Rupin Mittal.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/a1ecbd2950e6a24c6b256e3f96a4226941c07d96

* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/history-back-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/navigation-back-forward-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/navigation-navigate-replace-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/currententrychange-event/navigation-navigate-same-doc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/cross-origin-traversal-redirect.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-handler-throws.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-multiple-times-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-popstate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-after-detach.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-dynamic-index.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-noop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-navigation-navigate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-navigation-navigate.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-in-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-back-same-document-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-in-iframe-same-document-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-navigates-top-and-same-doc-child-and-cross-doc-child.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-same-document-preventDefault-multiple-windows.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigation-traverseTo-top-cancels-cross-document-child.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/replaceState-inside-back-handler.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/same-url-replace-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/same-url-replace-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-history-replaceState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-initial-about-blank.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-push-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-replace-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-same-document-then-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse-not-in-entries.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse-then-clobber.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/after-detach.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entries-after-blank-navigation-from-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/resources/opaque-origin-page.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/back-forward-multiple-frames.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/disambigaute-traverseTo-forward-multiple.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/forward-to-pruned-entry.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/navigate-history-push-same-url-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/navigate-history-push-same-url.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-already-detached.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-beforeunload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-intercept-rejected.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/back.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-already-detached.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-beforeunload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-intercept-rejected.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/forward.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-intercept-interrupted.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-interrupted-within-onnavigate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-interrupted.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-javascript-url-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-javascript-url.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-push-initial-about-blank.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-push-javascript-url-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/navigate-push-javascript-url.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/resources/back-forward-opaque-origin-page.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-cross-document-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-cross-document-before-navigate-event.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-same-document-before-navigate-event.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-detach-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-intercept-rejected.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo-repeated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/traverseTo.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/traverseTo-multiple-steps.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/traverseTo-navigates-multiple-iframes.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/anchor-download-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/anchor-download-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/back-same-document-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/back-same-document-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/back-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/currententrychange-before-popstate-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/intercept-async.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-canceled.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-double-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-intercept-reentrant.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/location-href-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-canceled.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-double-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-in-transition-finished.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-stop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reentrant.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-canceled.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-no-popstate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-options.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push-changed-to-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace-changed-to-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-throws.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-commit-new-navigation-before-commit.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-window-stop-before-commit.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-uncancelable.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-window-stop-before-commit.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/resources/helpers.mjs: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/resources/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-basic.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-change-history-scroll-restoration-during-promise.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-explicit-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-intercept-handler-modifies.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/after-transition-timing.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-basic.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-immediate-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-after-dispatch.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-after-resolve.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-fragment-does-not-exist.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-in-precommit-handler.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-push.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-repeated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-replace.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-resets-when-no-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/scroll-after-preventDefault.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/scroll-without-intercept.html:

Canonical link: https://commits.webkit.org/300203@main
